### PR TITLE
Normalized all Turtle files

### PR DESCRIPTION
**Pull request by junrongYu**
Normalize all Turtle files in main branch.

### CODE REVIEW

File: vocab_files/methods_by_module/camera-traps/collection.ttl [hunk 1]
```diff
@@ -303,21 +293,16 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
...
-            widteh="620"
-            height="548"
-          />
+          <img src="https://w3id.org/tern/static/dawe-rlp/latest/camera-traps/detection-fauna-camera-traps.png" alt="The detection of fauna by camera traps is affected by ecological and observational processes occurring" widteh="620" height="548"/>
         </td>
       </tr>
     </tbody>
   </table>
-</p>
+<p/>
```
Comment (edmondchuc) [inline vocab_files/methods_by_module/camera-traps/collection.ttl:305]: 🤔 is this valid?

File: vocab_files/methods_by_module/basal-area/basal-wedge-protocol/basal-wedge-protocol.ttl [hunk 2]
```diff
@@ -16,50 +16,45 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
...
             <tbody>
               <tr>
                 <td>
-                  <img
-                    src="https://w3id.org/tern/static/dawe-rlp/latest/basal-area/basal-area-3-wedges.png"
-                    alt="Examples of how to score trees using the TERN Basal Wedge"
-                    widteh="644"
-                    height="260"
-                  />
+                  <img src="https://w3id.org/tern/static/dawe-rlp/latest/basal-area/basal-area-3-wedges.png" alt="Examples of how to score trees using the TERN Basal Wedge" widteh="644" height="260"/>
```
Comment (edmondchuc) [inline vocab_files/methods_by_module/basal-area/basal-wedge-protocol/basal-wedge-protocol.ttl:57]: Typo: found `widteh`, expected `width`

**Review by edmondchuc — COMMENTED** [review]
I think `html5lib` it is producing some invalid HTML. `<p><p/>` and `<th><th/>` are some examples

File: vocab_files/methods_by_module/camera-traps/collection.ttl (hunk 1 above)
Comment (junrongYu) [reply vocab_files/methods_by_module/camera-traps/collection.ttl:305]: It's shown correctly on browser, but I'll change them back to </p>

**Pull request closed without merging**